shared-libs/transitions/src/lib/infodoc.js

```diff
@@ -37,12 +37,12 @@ const getInfoDoc = change => {
       });
     })
     .then(doc => {
-      if (doc) {
```

Comment:
in the old code, if you have no document then it just creates a new one without trying to set the `transitions` property from the existing document, which seems wrong to me.